src/index.js

```diff
@@ -99,8 +98,12 @@ function generateFnFor(rpcMethodName, methodObject) {
           }
         })
         .catch(error => {
-          const outputError = new Error(`[ethjs-query] while formatting outputs from RPC '${JSON.stringify(error, null, this.options.jsonSpace)}'`);
-          reject(outputError);
+          let errorMsg = 'RPC error';
+          if (error.value) {
+            errorMsg += error.value.code ? ` (code ${error.value.code})` : '';
+            errorMsg += error.value.message ? `: "${error.value.message}"` : '';
+          }
+          reject(new Error(errorMsg));
```

Comment:
Do we need to modify the error? I don't think we really need to do that here. The code will be included in the error message for debugging purposes. If we create a new error it confuses our reporting so we should simply handle the error and reject it (remove lines 101-105, and simply `reject(error)` on line 106

Reply:
It is also unclear currently that this is an RPC error, rather than an error caused by this library. I think it's better to be explicit `"RPC error: (code 32000) already known"` rather than `"{value: { code: 32000, message: already known}}"`

Reply:
Either way we cannot throw a new error or we will override the stack trace leading to our error reporting tools bundling very much different errors in the same bucket because it looks like it came from the same place. I tested my branch locally by copying the dist file over to the metamask-extension's node_modules to verify that the error made sense from a browser perspective. however its possible that my solution would be worse for dapp developers.